pages/auth/register.tsx

```diff
@@ -35,6 +35,11 @@ const RegisterScreen = () => {
       return;
     }
 
+    if (!validator.isStrongPassword(registerForm.password)) {
+      setErrorMsg('Seems like your password is not safe. A safe password must have:- at least 8 characters, 1 uppercase letter, 1 lowercase letter, 1 symbol (.,*-%)');
```

Comment:
Change the message to Spanish

Reply:
site is in english, does not require Spanish alerts